### ISSUE TITLE
mandatory sh:path in PropertyShape

### ISSUE THREAD

**Issue by bertvannuffelen** [w3c/data-shapes#156]
In https://www.w3.org/TR/shacl/#property-shapes

>
> [SHACL instances](https://www.w3.org/TR/shacl/#dfn-shacl-instance) of sh:PropertyShape have one [value](https://www.w3.org/TR/shacl/#dfn-value) for the property sh:path.
> 

The above statement enforces the need for a mandatory sh:path in a PropertyShape. However one can also use other path statements https://www.w3.org/TR/shacl/#property-paths.

Can the informal statement below be improved as follows:

>
>Informally, property shapes specify constraints that need to be met with respect to [nodes](https://www.w3.org/TR/shacl/#dfn-node) that can be reached from the [focus node](https://www.w3.org/TR/shacl/#dfn-focus-node) either by directly following a given property (specified as an [IRI](https://www.w3.org/TR/shacl/#dfn-iri)) or any other [SHACL property path](https://www.w3.org/TR/shacl/#dfn-shacl-property-path), specified using sh:path.
>
The shacl property path is expressed using the operations defined in 2.3.1.

**Comment by VladimirAlexiev** [comment]
@bertvannuffelen Yes, `sh:path` is mandatory. It can be an IRI or a SHACL property path. Where is the mistake in the first paragraph you quote?

> Can the informal statement below be improved

What improvement are you suggesting? Your quotation is verbatim from the spec.

**Comment by HolgerKnublauch** [comment]
Closing as the OP never responded further. Please reopen if you disagree.
